iothub/service/src/FeedbackBatch.cs

```diff
@@ -11,10 +11,19 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class FeedbackBatch
     {
+        /// <summary>
+        /// A timestamp that indicates when the feedback message was received by the IoT hub.
+        /// </summary>
         public DateTime EnqueuedTime { get; set; }
 
+        /// <summary>
+        /// A collection of feedback records across multiple devices and messages for an IoT hub.
```

Comment:
I would rather say -  A collection of feedback records of C2D messages across multiple devices. - to avoid confusion

Reply:
This was the official documentation. But I do see your point. I'll update it.